developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst

```diff
@@ -143,7 +143,7 @@ PHP 5.6 (supported)
 
     $ sudo apt-get update
     $ sudo apt-get install mongodb
-    $ sudo apt-get install php5-mongo
+    $ sudo apt-get install php-mongo
```

Comment:
Hmm. I did not try to to go the PHP 7.0 way, but there is plainly no package named php5-mongo in ubuntu 16.04.. Hence, it cannot be removed ;)
The following should be enough:
`sudo apt-get install php-mongo`